test/fixtures/config/config-with-include-paths.yml

```diff
@@ -0,0 +1,6 @@
+template: Placeholder with example. Automatically calculated values based on previous releases are next major=$NEXT_MAJOR_VERSION, minor=$NEXT_MINOR_VERSION, patch=$NEXT_PATCH_VERSION. Manual input version is $INPUT_VERSION.
```

Comment:
How about including `$CHANGES` into the template, I assume a PR made it into the changes block? 🤔

Properly better to use something like:

```
template: |
  # What's Changed
  $CHANGES
```

Reply:
That clicked what was missing about my tests, hah! Finally got the snapshot tester working as well, I actually believe this PR is good, other than style or structure nits. The functionality is there, I've tested it in my own test harness.

Reply:
Luckily style and structure is taken care by our js tooling as well 👏 eslint, prettier, lint-staged and husky 👏